### PR TITLE
feat(sns-cli): Rename `Distribution.InitialBalances.governance` as `Distribution.InitialBalances.treasury`.

### PR DESCRIPTION
**Pull request by aterga**
Currently, `Distribution.InitialBalances.governance` is being mapped to `initialTokenDistribution.treasuryDistribution.total`. 

With this change, `Distribution.InitialBalances.governance` will be renamed to `Distribution.InitialBalances.treasury`.

### CODE REVIEW

**Comment by aterga** [comment]
After this is merged, we need to also merge https://github.com/dfinity/sns-testing/pull/81/files

**Review by anchpop — APPROVED** [review]
Approved, as long as you also make the appropriate changes in sns-testing and in any documentation we have that references this field (if we had any, I think it would be in sns-testing or in https://github.comdfinity/portal)

**Pull request merged**
